src/styles/components/_form.scss

```diff
@@ -36,34 +38,63 @@
   &__input-label {
     display: block;
     margin-bottom: rem(12px);
+
+    font-size: rem(16px);
+    font-weight: 600;
+    line-height: 1.2;
   }
 
-  &__input-group {
+  &__group {
```

Comment:
According to this [ref](https://9elements.com/bem-cheat-sheet/), should we change it to `input-group`. The name of `element` inside `block` is more meaningful when we say `input-group` of `form` than `group` of `form` What do you think?

Reply:
Yes, I agree sir.